src/main/java/com/fismobile/appium/base/BaseClass.java

```diff
@@ -32,23 +32,21 @@
 public class BaseClass {
 
     private static WebDriver driver;
-    private TestContext testContext;
     private Logger logger = LogManager.getLogger(BaseClass.class);
 
-
-    private WebDriver setup()
+    @Parameters({"platform","runOn"})
+    @BeforeClass
+    public void setup(@Optional("web") String platform, @Optional("chrome") String runOn)
```

Comment:
All the strings should be defined as constants in a common file for a better use, reuse and maintenance.